config/locales/zh-CN.yml

```diff
@@ -377,28 +387,29 @@ zh-CN:
     admin:
       tab:
         configuration: "配置"
-        option_types:
+        option_types: "选项类型"
         orders: "订单"
         overview: "概览"
         products: "产品"
         promotions: "促销"
-        properties:
-        prototypes:
+        promotion_categories: "促销分类"
```

Comment:
It is used in sidebar menu of spree_backend, see https://github.com/spree/spree/blob/master/backend/app/views/spree/admin/shared/sub_menu/_promotion.html.erb#L3 . It is used as sub key in `admin/tab`, see https://github.com/spree/spree/blob/master/backend%2Fapp%2Fhelpers%2Fspree%2Fadmin%2Fnavigation_helper.rb#L30 .